README.md

```diff
@@ -142,16 +154,61 @@ best_it_contentful:
     type:     annotation
 ```    
 
-Just add the reset controller to your routing (_we suggest to protected it with a http auth password_) and input this
- url in your contentful webhook config and you are ready to go.
+To activate the webhooks in contentful follow the following and guide [guide](https://www.contentful.com/developers/docs/concepts/webhooks/)
+and configure the webhook with the following table:
+
+| Type  | Event   | URL    | Description                |
+|-------|---------|--------|----------------------------|
+| Entry | create  | /fill  | A entry has been created. The cached entry will be created.   |
```

Comment:
What happens if an entry is created but not published? Will it be cached and shown in storefront?

Reply:
I have added a new implementation for the cache fill controller. Only published objects will be cached in the live modus. In the preview mode every change will be cached